x-pack/solutions/security/plugins/security_solution/common/api/detection_engine/rule_monitoring/model/execution_metrics.schema.yaml

```diff
@@ -25,6 +25,10 @@ components:
           description: Duration in seconds of execution gap
           type: integer
           minimum: 0
+        frozen_indices_queried_count:
+          description: Count of frozen indices queried during the rule execution
```

Comment:
It's worth including the detail that we consider the index to be "queried" if we can't exclude the whole index using the time range filter

Reply:
What do you think about something like this? 👇 
`Count of frozen indices queried during the rule execution. These indices could not be entirely excluded after applying the time range filter`

Reply:
that looks good! i'd just add a period at the end too

Reply:
Done!